### PR TITLE
Shortcut fixes

### PR DESCRIPTION
**Pull request by AkalUstat**
* [x]  Passed [sanity tests](http://bit.ly/sttm-sanity-tests).
* [x]  Ran `npm test` & fixed newly introduced lint errors.
* [x]  Checked console for errors.

Fixes for shortcut:

- Search shortcut now clears the search bar
- using center align shortcut no longer opens the console

### CODE REVIEW

**Comment by AkalUstat** [comment]
is it possible that instead of repeating code we just have the action handlers take care of this?

**Comment by AkalUstat** [comment]
Ok so i have done added that functionality to shortcut and allocated the auto toggling nature to the reducer (and so have removed the manual toggling from the controls)

File: package.json [hunk 1]
```diff
@@ -126,4 +126,4 @@
       "pre-push": "npm run pre-push-message && npm run test"
     }
   }
-}
+}
```
Comment (plug-n-play) [inline package.json:129]: image
Please fix this

Comment (AkalUstat) [reply package.json:129]: what does this mean?

Comment (saintsoldierx) [reply package.json]: @AkalUstat at the end of the file, add a new empty line.

**Pull request merged**